build.sbt

```diff
@@ -43,6 +44,10 @@ lazy val commonSettings = Seq(
   scalacOptions in (Compile, doc) := (scalacOptions in (Compile, doc)).value.filter(_ != "-Xfatal-warnings")
 ) ++ warnUnusedImport
 
+lazy val machinistDependencies = Seq(
+  libraryDependencies += "org.typelevel" %%% "machinist" % "0.4.1"
```

Comment:
Ahh yes. Sorry. LGTM.

On Thu, Jan 14, 2016 at 2:23 PM, Rüdiger Klaehn <notifications@github.com>
wrote:

> In build.sbt <https://github.com/non/cats/pull/806#discussion_r49794859>:
>
> > @@ -43,6 +44,10 @@ lazy val commonSettings = Seq(
> >    scalacOptions in (Compile, doc) := (scalacOptions in (Compile, doc)).value.filter(_ != "-Xfatal-warnings")
> >  ) ++ warnUnusedImport
> >
> > +lazy val machinistDependencies = Seq(
> > +  libraryDependencies += "org.typelevel" %%% "machinist" % "0.4.1"
>
> Note that machinistDependencies are only used in cats-core, not in
> cats-kernel. I don't know if this can be compile-only, but I did not want
> to mess with things other than cats-kernel for this PR.
>
> cats-kernel does not have a dep on machinist for now. See pom.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/non/cats/pull/806/files#r49794859>.
>



-- 
P. Oscar Boykin, Ph.D. | http://twitter.com/posco | http://pobox.com/~boykin